.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: qK9CfT3KkPW251ZukOPIb986eIa6jjjIl
```

Comment:
I'm not familiar with the yml file for coveralls too much, but I did see it says repo token.  Double-checking this is OK to upload?

Reply:
This is fine. This was from my PR the other day but I'm not sure what's going on here with the commit history.

Reply:
I rebased again from the correct branch, which should fix the commit issues.

Reply:
Looks like the commit history still isn't correct after rebasing. :sob: